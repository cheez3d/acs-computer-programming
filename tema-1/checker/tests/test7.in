Paris Londra Roma Moscova
18882555#555338#820177775566675330188844455566444887777
Capitala Islandei este Reykjavik
2
Islandei Elvetiei
Reykjavik Berna
atena berlin praga roma
