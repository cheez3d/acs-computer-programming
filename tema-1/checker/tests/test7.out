172777444777701555666#663777201777666#62016#66677772226668882
Valletta Skopje Vilnius
1222#274448255520133555888338444334440337777833012233777662
4
2833662 223377755544466 7777242 77766662 
223377755544466 2
0
0
